tools/README

```diff
@@ -103,6 +103,35 @@ Examples:
   $ sof-logger -l ldc_file -i trace_dump -o out_file -c 19.9
 
 
```

Comment:
@abonislawski it is not very clear to me why there are two README files? tools/README and tools/coredumper/README. Can you just have one?

Reply:
deleted one